python/lsst/pipe/tasks/finalizeCharacterization.py

```diff
@@ -477,6 +483,12 @@ def _make_selection_schema_mapper(self, input_schema):
             size=10,
             doc="Color used in PSF fit."
         )
+        selection_schema.addField(
+            'psf_max_value',
+            type=np.float32,
+            doc="PSF max value.",
```

Comment:
Could you please update these docs to be not just the fieldname? Something like "Peak value of the PSF model" or "Maximum value in the PSF/star image" for example, as appropriate?